consensus/istanbul/core/commit_test.go

```diff
@@ -36,6 +38,9 @@ func TestCore_sendCommit(t *testing.T) {
 
 	mockCtrl.Finish()
 
+	// set hardForkBlockNumberConfig
+	fork.SetHardForkBlockNumberConfig(&params.ChainConfig{IstanbulCompatibleBlock: istanbul.TestBlockHF})
```

Comment:
It needs to be back to the original HardforkBlockNumber also.

Reply:
Reflected in new change https://github.com/klaytn/klaytn/pull/997/commits/7b268b64f681d73f4839a0569baa2484aad823b3